### DISCUSSION TITLE
Add cosmetic changes

### DISCUSSION THREAD

**Discussion by X-Anonymous-Y** [tteck/Proxmox#2075] [Ideas]
Hey @tteck,
Would you be interested on cosmetic changes?

before:
```
HEADER
...
Using Disk Size: 4
Allocated Cores: 2
Allocated RAM: 1024
...
```

after: changed Cores text, added automatically detects (Core/s), added named values ​​for Disk, Core/s and RAM
```
HEADER
...
Using Disk Size: 4 GB
Allocated CPU: 1 Core
...
Allocated CPU: 4 Cores
Allocated RAM: 1024 MB
...
```

it's just an idea

**Comment by tteck** [comment]
~You realize that's around 120 files to modify for  "cosmetic changes", I'll pass.~
1 file, maybe

**Reply by X-Anonymous-Y** [reply]
no!
This only requires changes in "function echo_default()" and "function advanced_settings()" as well as in scripts that are not installed via "build.func"
